src/components/InputPreview.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { ChangeEvent, useState } from "react";
+import { Button } from "./atomic";
+
+type PreviewInput = {
+  setFile(file: File | null): void;
```

Comment:
nit:

setFile: (file: File | null) => void; 

is more readable and indicative of a function in my opinion